webapp/client/public/css/assorted.css

```diff
@@ -424,9 +419,27 @@ footer a{
   margin-top: var(--spacing-09);
 }
 
+.section-intro {
+  margin-top: var( --spacing-11);
+}
+
+.header-navigation, .header-navigation + .section-intro {
```

Comment:
This will check if a header-navigation exist before the section-intro. if so reduce the margin top to give the navigation some space

Reply:
this is not the case on the vorbereiten subpages . eg. /vorbereiten/bestattungskosten we have the header navigation present, but no .section-intro, so the margin top is still spacing-11 instead of spacing-07. Correct?

Reply:
Fixed that! thank you tarek